### PR TITLE
Add gem code

### PR DESCRIPTION
**Pull request by AnnaKL**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+gem 'reliable-queue-rb', '~> 0.1.0'
+```
+
+And then execute:
+
+    $ bundle
+
+Or install it yourself as:
+
+    $ gem install identifiers
```
Comment (jbilbo) [inline README.md]: Change this to: gem install reliable-queue-rb

File: travis.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+language: ruby
```
Comment (jbilbo) [inline travis.yml]: This filename is missing the first dot (`.`) character: `.travis.yml`

Comment (AnnaKL) [reply travis.yml]: oh good catch 👍

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+language: ruby
+rvm:
+  - 2.0
```
Comment (jbilbo) [inline .travis.yml]: Let's leave only 2.5 to 2.7 (the current maintained ones)

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+  - 2.1
+  - 2.3
+  - 2.3
+  - 2.4
+  - 2.6
+  - 2.7
+before_install:
+   sudo service redis-server start --bind 127.0.0.1
```
Comment (jbilbo) [inline .travis.yml]: I think we need something like this instead:
```
services:
  - redis-server
```
See: https://docs.travis-ci.com/user/database-setup/#redis

Comment (AnnaKL) [reply .travis.yml]: This worked and builds started passing. Should I change it anyway?

**Review by jbilbo — APPROVED** [review]

**Pull request merged**